### ISSUE TITLE
Not using entire dataset for finetuning

### ISSUE THREAD

**Issue by adnan-hossain** [LuizScarlet/StableCodec#14]
Hi! Great work! I just wanted to clarify something. 

With ```--max_train_steps=21000``` for finetuning and ```--train_batch_size=1```, it seems like you do not finetune over the whole training dataset (less than 1 epoch of training). Am I correct? and is there a reason why you do this?

**Comment by LuizScarlet** [comment]
Hi @adnan-hossain , by default we use 2x RTX 3090 GPUs for training and a gradient accumulation step of 4 on each GPU. The mini batch size per GPU is set to 1 using `--train_batch_size=1`, so the actual batch size is 8 in total.
